apiary.apib

```diff
@@ -2637,6 +2639,81 @@ enabled. See <a href="/#reference/0/users/create-two-factor-authentication-secre
             "Message": ""
         }
 
+### Login with token [POST /organizations/auth/token]
+
+Login with a user token
```

Comment:
Please provide more information about the user token. If login with the user token obtained from /user/login EP, the response token will still be able to make measurements. It's suggested to update to be the following:

Login with a user token obtained from <a href="/#reference/0/organizations/login">Login</a> endpoint.

Reply:
updated